diff_general/run_tests.py

```diff
@@ -9,4 +9,5 @@
         klass = build_test_class(name)
         suite.addTest(klass(TestMetaclass.test_name(name)))
 
-    JSONTestRunner(visibility='visible').run(suite)
+    with open('/autograder/results/results.json', 'w') as f:
```

Comment:
I just copied the path from `run_autograder`. It's exactly the same. How would it make it difficult on the local machine? Do you not run `run_autograder` locally?

Reply:
Actually, I got this code from some of your other examples.

Reply:
Right, it is used in some other examples. But if you don't have an `/autograder` directory on your local machine this line may throw an error trying to open that file IIRC?

Reply:
Yes, agreed, but I was confused on how that wasn't the case before. But, I guess in the original case, it was easy to remove the redirect. Is that what you were thinking also?

Reply:
Oh I was running `run_tests.py` directly rather than `run_autograder` to see the output. But actually, I'm also working on something to make it easier to actually run the Docker container locally, but that would require using Docker, so it'd still be better to have a way to run it locally to see how things are working.